src/Autodesk.Forge.DesignAutomation/Model/StringArgument.gen.cs

```diff
@@ -41,10 +41,15 @@ public partial class StringArgument : IArgument
         /// <summary>
         /// Initializes a new instance of the <see cref="StringArgument" /> class.
         /// </summary>
+        public StringArgument(string v) : base()
+        {
+            this.Value = v;
+        }
+
         public StringArgument() : base()
         {
         }
-        
+
         /// <summary>
```

Comment:
not exactly the same as `StringArgument` at front-end. which doesn't have constructor `public StringArgument()`, but I think it should be ok to keep it